defaults/main.yml

```diff
@@ -5,7 +5,7 @@ nrinfragent_service_state: "started"
 nrinfragent_config:
   license_key: YOUR_LICENSE_KEY
 nrinfragent_integrations: []
-nrinfragent_license_key: null
+#nrinfragent_tarball_version: "1.3.27"
```

Comment:
I'd remove this, as we will need to keep upgrading the version on every release, the option is documented, so there is no need to put it as a comment in the defaults